cmd/fluent-watcher/fluentbit/VERSION

```diff
@@ -1 +1,2 @@
+# renovate: datasource=docker depName=ghcr.io/fluent/fluent-operator/fluent-bit version=
```

Comment:
is that `version=` intented at the end?

@joshuabaird Please note this is now gonna cause a release, so ensure to cancel the workflow straight after merge, that is why I doubted if we should already merge that.